crates/core/src/test_utils/bin/dump-katana.rs

```diff
@@ -24,43 +24,40 @@ async fn main() {
         })
         .await;
 
-    tokio::task::spawn_blocking(move || {
-        // Get a serializable state for the sequencer
-        let sequencer = test_context.sequencer();
-        let dump_state = sequencer
-            .sequencer
-            .backend
-            .state
-            .blocking_write()
-            .dump_state()
-            .expect("Failed to call dump_state on Katana state");
+    // Get a serializable state for the sequencer
+    let sequencer = test_context.sequencer();
+    let dump_state = sequencer
+        .sequencer
+        .backend
+        .state
+        .write()
+        .await
+        .dump_state()
+        .expect("Failed to call dump_state on Katana state");
 
-        let state = serde_json::to_string(&dump_state).expect("Failed to serialize state");
+    let state = serde_json::to_string(&dump_state).expect("Failed to serialize state");
 
-        // Dump the state
-        std::fs::create_dir_all(".katana/").expect("Failed to create Kakata dump dir");
-        std::fs::write(".katana/dump.json", state).expect("Failed to write dump to .katana/dump.json");
+    // Dump the state
+    std::fs::create_dir_all(".katana/").expect("Failed to create Kakata dump dir");
+    std::fs::write(".katana/dump.json", state).expect("Failed to write dump to .katana/dump.json");
 
-        let deployer_account = DeployerAccount {
-            address: test_context.client().deployer_account().address(),
-            private_key: *STARKNET_DEPLOYER_ACCOUNT_PRIVATE_KEY,
-        };
+    let deployer_account = DeployerAccount {
```

Comment:
what's the difference between this `_Deployer_Account` and any starknet account? is no, why a _Deployer_ then?